span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/SpanProcessingConfigServiceImpl.java

```diff
@@ -67,10 +104,35 @@ public void getAllExcludeSpanRules(
     try {
       RequestContext requestContext = RequestContext.CURRENT.get();
       this.validator.validateOrThrow(requestContext, request);
+      List<ExcludeSpanRuleDetails> excludeSpanRules =
+          excludeSpanRulesConfigStore.getAllData(requestContext);
+
+      Map<String, ExcludeSpanRuleDetails> tenantLevelExcludeSpanRulesToIdMap =
```

Comment:
Instead of tenant level, can we call them user?
```suggestion
      Map<String, ExcludeSpanRuleDetails> userExcludeSpanRuleIdToRuleMap =
```

Though, I don't believe we even need to build the map